data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -161,7 +161,7 @@ private static Metrics getAvroMetrics(Path path, Configuration conf) {
     }
   }
 
-  private static Metrics getParquetMetrics(
+  public static Metrics getParquetMetrics(
```

Comment:
is it possible to leverage `ParquetUtil.fileMetrics` directly, instead of making these public methods?

Reply:
Similar comment for Avro and ORC

Reply:
Thank you for your suggestion. It seems doing so would mean copy-pase the `getParquetMetrics` from `TableMigrationUtil` to `BaseMigrateDeltaLakeTableAction`. I think the choice here is between duplicated code section and exposing private methods. Given that the class named `TableMigrationUtil` is intended to provide util methods for table migration, do you think it may be proper to make these methods public in this case?